tests/main/test_integration.py

```diff
@@ -439,6 +440,8 @@ def test_system_states_and_recorded_metadata_with_update_to_file_writer_director
         # Tanner (6/15/20): Processes must be joined to avoid h5 errors with reading files, so clearing queues manually here in order to join
         test_process_manager.hard_stop_and_join_processes()
 
+        well_def = LabwareDefinition(row_count=4, column_count=6)
```

Comment:
note for future - this can be declared at the top of the file as a module-level constant (as presumably it will be useful in other tests as well)---or maybe even as a constant in `fixtures.py` that could be imported across multiple tests.  maybe even within the desktop app repo `constants.py` itself. maybe even in the future it could be worthwhile having a few constants declared in the `labware-domain-models` package itself for immutable versions of very common formats like 24 well and 96 well and 384 well plates